test/runtests.jl

```diff
@@ -112,7 +112,6 @@ include("Experimental/JuLie-test.jl")
 include("Experimental/SymmetricIntersections-test.jl")
 include("Experimental/ExteriorAlgebra-test.jl")
 include("Experimental/LinearQuotients/runtests.jl")
-include("Experimental/FTheoryTools/runtests.jl")
```

Comment:
How will the tests for `FTheoryTools` be executed in the future? Is this in preparation of a possible split of the CI-jobs into `core` and `experimental`?

Reply:
[`test/runtest.jl`](https://github.com/oscar-system/Oscar.jl/blob/6cf96ad70320645093eacdcd8ae9d1cef22cf7e4/test/runtests.jl#L101-L103) calls [`experimental/runtests.jl`](https://github.com/oscar-system/Oscar.jl/blob/6cf96ad70320645093eacdcd8ae9d1cef22cf7e4/experimental/runtests.jl) which in turn calls the testfile of FTheoryTools based on some regex.